access/slack/bot.go

```diff
@@ -268,7 +273,11 @@ func (b Bot) msgSections(reqID string, reqData RequestData) []BlockItem {
 		msgFieldToBuilder(&builder, "Role(s)", strings.Join(reqData.Roles, ","))
 	}
 	if reqData.RequestReason != "" {
-		msgFieldToBuilder(&builder, "Reason", reqData.RequestReason)
+		// Slack has a 4000 character recommendation (with a more generous 40000
```

Comment:
Rather than adding duplicates and pointers to this comment, can you make 500 a constant and move the comment to that constant? Maybe make it a separate constant for slack and mattermost as well.

Reply:
Added commented constants for both.